ocw/models.py

```diff
@@ -56,9 +57,9 @@ class Instance(models.Model):
     instance_id = models.CharField(max_length=200)
     region = models.CharField(max_length=64, default='')
     vault_namespace = models.CharField(max_length=64, default='')
-    csp_info = models.TextField(default='')
     notified = models.BooleanField(default=False)
     ignore = models.BooleanField(default=False)
+    IGNORE_TAG_NAME = 'pcw_ignore'
```

Comment:
```suggestion
    TAG_IGNORE = 'pcw_ignore'
```
If we track another tags, I believe it's much better for these constants to have the `TAG_` prefix.